### ISSUE TITLE
Club maker shows Area Number but no 'letter' for the division

### ISSUE THREAD

**Issue by stabbystabstab** [stabbystabstab/district-club-map#5]
For accessibility reasons, clubs should be labelled with the full Toastmasters Area name, id 'C7' or 'M22' rather than just '7' or '22'

**Comment by stabbystabstab** [comment]
Done
